go.mod

```diff
@@ -2,22 +2,22 @@ module github.com/ipfs/go-ipfs-http-client
 
 require (
 	github.com/ipfs/go-block-format v0.0.3
-	github.com/ipfs/go-cid v0.0.7
-	github.com/ipfs/go-ipfs-cmds v0.6.0
-	github.com/ipfs/go-ipfs-files v0.0.8
+	github.com/ipfs/go-cid v0.2.0
+	github.com/ipfs/go-ipfs-cmds v0.8.1
+	github.com/ipfs/go-ipfs-files v0.1.1
 	github.com/ipfs/go-ipld-format v0.4.0
 	github.com/ipfs/go-merkledag v0.6.0
-	github.com/ipfs/go-path v0.1.1
-	github.com/ipfs/go-unixfs v0.2.5
+	github.com/ipfs/go-path v0.3.0
+	github.com/ipfs/go-unixfs v0.3.1
 	github.com/ipfs/interface-go-ipfs-core v0.7.0
 	github.com/ipfs/iptb v1.4.0
-	github.com/ipfs/iptb-plugins v0.3.0
-	github.com/libp2p/go-libp2p-core v0.8.6
+	github.com/ipfs/iptb-plugins v0.3.1-0.20220623115941-19e323570cb1
+	github.com/libp2p/go-libp2p-core v0.17.0
```

Comment:
Looks like this package is deprecated now
https://github.com/libp2p/go-libp2p-core#deprecation-notice